src/Wanderer.html

```diff
@@ -3538,6 +3544,33 @@
  document.getElementById("statusLine").innerHTML = names+"<br />"+attributes+"<br />"+equipped+"<br />"+states+msg;
 }
 
+function mapTreasureOnto(outputMap) {
+ // treasure your Self first
+ var treasure = [[userPos.x, userPos.y]];
+ var legend = impassable.concat(['$', tileKey, ']', tileArmor, tileStaff, tileTreasure, '?']);
+ for (var i = 0; i < mapWidth; i++) {
+  for (var j = 0; j < mapWidth; j++) {
+   if ('$' == outputMap[i][j]) {
+    treasure.push([i,j]);
+   } else if (!legend.includes(outputMap[i][j])) {
+    outputMap[i][j] = emptyBlock;
+   }
+  }
+ }
+ // Draw a perimiter within which treasure is worth a round trip
+ for (var t=treasure.length-1; t >= 0; t--) {
+  var tx = treasure[t][0];
+  var ty = treasure[t][1];
+  for (var i = 0; i < mapWidth; i++) {
+   for (var j = 0; j < mapWidth; j++) {
+    if (stepsTo(tx, ty, i, j) == 10 && !legend.includes(outputMap[i][j])) {
+     outputMap[i][j] = t == 0 ? '+' :'˙';

```

Comment:
The `˙` character probably doesn't render all that nicely on all platforms/browsers. 
Definitely didn't in a Jupyter notebook.
<img width="1081" alt="Screen Shot 2020-08-15 at 5 49 56 PM" src="https://user-images.githubusercontent.com/292802/90322229-c593cf00-df1f-11ea-9795-242d33cfc9be.png">


Although `˙` looks nicer, a simple period `.` is probably a more portable choice:

<img width="1081" alt="Screen Shot 2020-08-15 at 5 51 51 PM" src="https://user-images.githubusercontent.com/292802/90322242-13103c00-df20-11ea-80fe-658ba00b6266.png">


```suggestion
     outputMap[i][j] = t == 0 ? '+' :'.';

```